docs/src/main/paradox/validation.md

```diff
@@ -53,3 +53,11 @@ paradoxValidationIgnorePaths ++= Seq(
   "/docs/version/(?!latest).*"
 )
 ```
+
+## Retrying links check
+
+`paradoxValidateLinksRetryCount` setting allows retrying link check for the provided number of times in case link returns non success response code i.e., other than 200 status code. 
```

Comment:
```suggestion
`paradoxValidateLinksRetryCount` setting allows retrying link check for the provided number of times in case linked server does not reply, or returns a possibly temporary failure response code (500, 502, 503 or 504). 
```

Reply:
Fix pushed.